nautobot/docs/user-guide/feature-guides/getting-started/creating-devices.md

```diff
@@ -73,25 +73,46 @@ To create a new Device Type:
 
 ![Create a device type](../images/getting-started-nautobot-ui/4-create-device-type.png)
 
-## Creating a Site
+## Creating a Location Type
 
-How you choose to employ sites when modeling your network may vary depending on the nature of your organization, but generally a site will equate to a building or campus.
-For example, a chain of banks might create a site to represent each of its branches, a site for its corporate headquarters, and two additional sites for its presence in two co-location facilities.
+To create a location, you need to specify a Location Type for the location.
 
-More information on Sites is found in the [Sites](../../core-data-model/dcim/site.md#sites) section of the Nautobot documentation.
+More information on Location Type is found in the [Locations](../../core-data-model/dcim/locationtype.md#location-types) section of the Nautobot documentation.
 
-To create a new Device, you will need an existing Site or need to create a new Site instance.
+To create a new Location, you will need an existing Location Type or need to create a new Location Type instance.
 
-To create a new Site:
+To create a new Location Type:
 
 1. Click on **Organization** in the top navigation menu
-2. Find **Sites**
-3. Select **+** to go to the `Add a new site` form
-4. Populate the Site's `Name`
-5. Set the `Status` to `Active` in the drop-down selector
-6. Click on `Create` at the bottom of the form (not shown)
+2. Find **LocationTypes**
+3. Select **+** to go to the `Add a new location type` form
+4. Select a `Parent` LocationType if needed
+5. Populate the LocationType's `Name`
```

Comment:
For completeness, would also want/need to specify the content-types that can belong to this location-type.